### ISSUE TITLE
Repubblica.it not working anymore

### ISSUE THREAD

**Issue by Heliosmaster** [gall0ws/freeRep#14]
Looks like the site has removed the old amp pages, so freeRep doesn't work anymore with it.

**Comment by gall0ws** [comment]
Thanks for the heads up!

Nevertheless, I found that it's still possible to bypass it:

```
$ check () {
     grep -q nomenklatura || {
 echo paywall
 return
     }
     echo ok
 }
$ url='https://www.repubblica.it/cultura/2021/12/03/news/renzo_piano_nel_mio_nuovo_museo_c_e_tutta_la_luce_di_mosca_-328839338/'
$ curl -s $url | check
paywall
$
$ curl -s $url -H 'Referer: https://translate.google.com/' | check
ok
$ curl -s $url -H 'Referer: https://m.google.com' | check
ok
$ curl -s $url -H 'Referer: https://www.google.com/' | check
ok
$ curl -s $url -H 'Referer: https://www.twitter.com' | check
paywall
$ curl -s $url -H 'Referer: https://www.bing.com' | check
paywall
```

So, as long as you came from google it's okay for them :shrug: 

Let's see if we can do that inside freeRep.

**Comment by gall0ws** [comment]
Released v1.6 that implements the referer trick for repubblica.it.
